dingomata/cogs/text.py

```diff
@@ -66,6 +66,11 @@ def render(self, **kwargs) -> str:
         template = self.templates.choose()
         return template.format(**fragments, **kwargs)
 
+    def renderSpecificAdv(self, bodyPart: str, **kwargs) -> str:
+        fragments = {k: v.choose() for k, v in self.fragments.items()}
+        template = self.templates.choose().replace("{bodyPart}", bodyPart)
+        return template.format(**fragments, **kwargs)
```

Comment:
We shouldnt need to add a separate function. Simply changing the existing function to something like `.format(**{**fragments, **kwargs})` should be sufficient, and that lets you pass whatever you need in, instead of only working for bodyPart alone